Interop/JsRuntime/src/main/kotlin/jsinterop.kt

```diff
@@ -26,9 +42,13 @@ external public fun pushIntToArena(arena: Arena, value: Int)
 
 const val upperWord = 0xffffffff.toLong() shl 32
 
+@Used
+@ExportForCppRuntime
```

Comment:
Shalln't @ExportForCppRuntime imply @Used?

Reply:
No but these functions are not called from JS so we still can remove `@Used` here.